### PR TITLE
Added testcase file

### PR DESCRIPTION
**Pull request by Loakeshvaran-SK-ML**

### CODE REVIEW

File: parsers/github/dockerfile
Comment (Dannana-Venkataramana-ML) [inline parsers/github/dockerfile:1]: Why is this file needed?

**Comment by Dannana-Venkataramana-ML** [comment]
Please attach the test coverage report

File: parsers/github/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ ENV PASSWORD=$PASSWORD
 COPY go.mod ./
```
Comment (Dannana-Venkataramana-ML) [inline parsers/github/Dockerfile:16]: Please make use of .dockerignore file to exclude unwanted files to copy in docker image which will eliminate multiple Copy commands in Dockerfile

File: parsers/github/dockerfile
Comment (Loakeshvaran-SK-ML) [reply parsers/github/dockerfile:1]: Removed the file

File: parsers/github/Dockerfile (hunk 1 above)
Comment (Loakeshvaran-SK-ML) [reply parsers/github/Dockerfile]: Added .dockerignore dile

File: parsers/github/main_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"encoding/json"
+	"sync"
+	"testing"
+	"time"
+	"errors"
+	"github.com/rabbitmq/amqp091-go"
```
Comment (Dannana-Venkataramana-ML) [inline parsers/github/main_test.go:9]: remove duplicate import

Comment (Loakeshvaran-SK-ML) [reply parsers/github/main_test.go]: Removed the duplicate import

**Review by Dannana-Venkataramana-ML — APPROVED** [review]

**Comment by Loakeshvaran-SK-ML** [comment]
image
